examples/knative/powershell/kn-ps-echo/test/send-cloudevent-test.ps1

```diff
@@ -4,11 +4,11 @@ $headers = @{
     "ce-specversion" = "1.0";
     "ce-id" = "id-123";
     "ce-source" = "source-123";
```

Comment:
nit: we could make the source used across the examples/tests here a proper source, e.g. `https://vcenter.local/sdk`